### ISSUE TITLE
Monthly Ticket for Quest Changes

### ISSUE THREAD

**Issue by chochem** [GTNewHorizons/GT-New-Horizons-Modpack#12495]
### Your GTNH Discord Username

chochom#0271

### Your Pack Version

2.2.9-9-pre-dev - 2.3.0-RC3

### Your Server

SP

### Type of Server

Single Player

### Overview

A ticket for any large or small problems with any quests. Just post below. Of course, you can also make a dedicated ticket. In particular for more indepth suggestion or problems.

The most up-to-date quest files can always be found here: https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/tree/master/config/betterquesting/DefaultQuests

Below are the remaining things from last month (https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/11981):

### Quest Changes

- [x] **Quest.1305 For Fluid Recipes**

  It's better to mention that "AE2FC do fluid recipes better" or something in this quest, imo. EC2 is slow and buggy.

- [ ] https://github.com/GTNewHorizons/GT5-Unofficial/pull/1146 reworked turbine materials, so quests related to turbine materials need to be checked to confirm if any changes are needed.

- [ ] **Quest.3165 UV Power Per Turbine**
  > Large Ichorium 170% 342,720 EU/t (very durable)

  Ichorium turbine has been buffed. So this data is outdated.

- [ ] **Quest.3166 Powering-up Fluid Nukes**
  > Oriharukon and MAR-Ce-M200 are your best choices.

  MAR-Ce-M200 is LuV (need ZPM tier EBF) now. See https://github.com/GTNewHorizons/GoodGenerator/blob/33681ff5f240800c70b70c1e314d73ae262caf7c/src/main/java/goodgenerator/loader/RecipeLoader_02.java#L1145

- [ ] **Quest.2568 Infinity Helmet** and **Quest.2569 Infinity Breastplate**
   Infinity Helmet can't remove all negative debuffs, it's the ability of Infinity Breastplate.

- [x] This quest: <details> <summary>pic</summary>![image](https://user-images.githubusercontent.com/19243993/216631625-44786d2b-d418-44fb-8f73-bfe7e323d7bb.png) </details>
This is incorrect; you get one unique bag as a reward for each piece of Kami armor, and another for the LEG. You can get 5 unique bags even without trading once. (Unless this is a typo and the sentence was supposed to start with "*Note* that...") -- Probably was just mistakenly added to LEG.

### New Quests/Full Overhauls

- [x] More AE2FC quests. replace old EC ones.

- [ ] Add a quest for Quadruple Input Hatch, or mention it in some multiblock quests (for example, EBF).

- [ ] Add a quest in Multblock Goals tab listing what GT tools can do with multiblock controllers. (Or add them to each multi's tooltip or quest) Examples are as follows:
  - Screwdriver: (Right Click) Toggle mode for some multis, lock recipe for LCR, set prioritise circuit for MEBF, toggle input bus seperation for PA, EBF, Industrial Wiremill etc. (Shift Right Click) Lock recipe for PA
  - Soldering Iron: (Right Click) Toggle void excess for GT++ multis
  - Wire Cutter: (Right Click) Toggle UEV+ treated as UHV for PA, toggle input bus seperation for AdvEBF, MEBF, ABS, etc.
  - Plunger: (Right Click) Clear output hatch for GT++ multis

- [ ] Add a quest for Integrated Ore Factory in Multiblock Goals tab (maybe add a link in UHV tab as well). It is the ultimate solution for ore processing in late game.

- [ ] Add a quest for Tesla Tower in Multiblock Goals tab. It is a powerful power transmission option in HV~ZPM that should be mentioned in questbook.

- [ ] Add a quest/quests for YOTtank in Multiblock Goals tab. It is the ultimate fluid storage solution in middle & late game.

- [ ] Add a quest for Infinite Oil/Gas/Fluid Drilling Rig. There is already a quest for void miner, so add one for Infinite Rig is reasonable.

- [ ] Add a quest for GT++ Solar Tower in How to Generate Power tab, it is now a power generation option in IV, since https://github.com/GTNewHorizons/GTplusplus/pull/290 fixed and reworked it.

- [ ] Add XL Gas Turbine and XL Plasma Turbine quests in How to Generate Power tab, they are good upgrade for the large ones. XL Steam/HP Steam Turbine have a quest in Working with Oil tab already, might need a link in How to Generate Power tab too?

- [ ] Add a quest for Turbine Housing in How to Generate Power tab, it makes automating large turbine possible.

- [ ] Add a quest for Elemental Duplicator in GT++ tab.

- [ ] Add quests for the Algae Farm. -- Regarding this point, the Algae Farm doesn't just need a quest, it needs an entire questline where it's properly balanced. It can replace several early automations for specific chemicals if used, so it is my (steelux) plan to make that questline when I finish the remaining adjustments.

- [ ] Quest.1510 Biomes Everywhere, But Not the One You Want? Overhaul based on https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/11385#issuecomment-1257110174

- [ ] add quests for super glue

- [ ] add quest for the component assembly

- [ ] add quest for advanced assembling line 

- [x] add quest for the QFT

- [ ] the Lutetium quests need an overhaul. Specifically, 'Faster Lutetium Excitement', 'High Temperature Super Breeder'. The neutron activator option was removed in favor of the HTGR.

- [ ] rewrite the TFFT quest(s) because of https://github.com/GTNewHorizons/KekzTech/pull/45

- [ ] add a stone age quest about visual prospecting (best also mention the spawn chunks bug)?

- [ ] add quests for bastline?

- [ ] add quests for th232 line in IV and/or ZPM alternative

- [x] add Logistics Pipes questline, tiered for MV and with a guide on how to make it work

**Comment by Quarri6343** [comment]
"Grinding With Wind" quest is outdated after the windmill rework
https://github.com/GTNewHorizons/bartworks/pull/205

**Comment by chochem** [comment]
@Quarri6343 any proposal of what it should say now?

**Comment by Quarri6343** [comment]
・"It's probably not actually worth it to make or use" should be removed
・"It's pretty slow~" is not true (compared to steam macerator)
・"You will only get one crushed Ore ~" is not true, you sometimes get 2 ores from it
・following descriptions should be added into the quest
->Speed and output will be affected by wind speed, recipe and rotor
->Macerates 16 items at a time
->High wind speeds will decrease output
->The durability will decrease according to the wind speed

**Comment by chochem** [comment]
- [ ] too many keys quest: most stuff is unbound now. Check and bind it yourself. Except for NEI keys, those are still set, point to the separate menu!

**Comment by AmberSherbet** [comment]
The Bricked Blast Furnace quest still has a warning encouraging players to make things one at a time before beginning the next step (in Dev 2.2.9-9, at least), when in reality the steps before the controller itself are now optional, and it would actually be faster and better to do multiple steps at once now. Suggest to remove the warning.

**Comment by vovaksenov** [comment]
It should be decided if the europium quest is to be separated from fusion T1 as you don't need fusion for europium anymore #12416

**Comment by chochem** [comment]
well that should be decided first before we change the quest

**Comment by King65868** [comment]
The UIV circuit assembler now has a recipe so the quest Magneto Resonatic Circuits should remove the reference that it isn’t craftable. 
The Circuit Assembly Line quest says it says you save soldering alloy when using it, but it also saves indalloy now.

**Comment by ksd77** [comment]
Hill Cherry #2128:
Slight typo, retrieval should be on the cherry sapling, not silver lime.
![изображение](https://user-images.githubusercontent.com/124521465/216835422-9501b8e4-f66c-4cb9-a04e-53b6fde32569.png)

**Comment by Ethryan** [comment]
- [ ] Make the steam age tinkers quest for the reinforcement modifier require the new item instead of the Large Obsidian Plate that it currently wants.

**Comment by AbdielKavash** [comment]
The "Control Units" quest in the Advanced Automation tab (beginning of the OC line) accepts any MV circuit, but needs specifically the basic Transistor. It would be nice if it could accept SMD transistors as well.

https://i.imgur.com/eghSmQj.png

**Comment by chochem** [comment]
just going through discord a bit. some people dont get why github is so important sadly:

- [ ] ![image](https://user-images.githubusercontent.com/40274384/218310309-d8053c02-ac38-4a69-84d0-5ab6b9988da5.png)

- [ ] ![image](https://user-images.githubusercontent.com/40274384/218310319-03b811e5-a63c-47a7-9f6e-8b5db11aee71.png)
![image](https://user-images.githubusercontent.com/40274384/218310325-1b9b2d5e-6aaa-42cf-8fe3-6392196d514b.png)

- [ ] ![image](https://user-images.githubusercontent.com/40274384/218310360-3de472e2-baba-4a74-abb7-043a3cbd0b5e.png)

- [ ] ![image](https://user-images.githubusercontent.com/40274384/218310411-19c49123-5874-4986-a960-a3c4fbd3ece5.png)

**Comment by AbdielKavash** [comment]
The "Better Question?" quest in Tips and Tricks details the various `bq_admin` commands. Unfortunately, the one that is probably the most relevant for the average player (`/bq_admin default load`) is somewhere in the middle where it is easily overlooked.

I suggest moving this command to the top of the quest, right after the first paragraph. I would also simplify the description to something like "Synchronize the questbook with your current modpack version. Use this command after you update the pack, or if you can't see some quests. This will not erase any of your quest progression."

**Comment by chochem** [comment]
or just sopy the default load quest there as well I guess

**Comment by chochem** [comment]
oh wait, its already there.

**Comment by AbdielKavash** [comment]
The Cleanroom quest is in a poor state. (Or well, the cleanroom *code* is in a poor state, but that is not the point here.)

* It mentions that a door is always needed. This is not the case. A cleanroom forms and works fine without a door.
* It says "Up to 5% of the plascrete blocks can be replaced with Reinforced glass", which while close, is not exactly how the math works. This came up in discord recently when the count was off by a small amount.
* More importantly, it doesn't mention at all that there are other valid types of glass, with much more generous limits: Borosilicate Glass (50% of plascrete), Borosilicate Glass (any other color or tier than the basic one) (100% of plascrete), or Warded Glass (50% of plascrete). At least Borosilicate Glass should be mentioned, it is much more convenient and less restricted in block count than reinforced glass.

This is valid (although extreme): https://i.imgur.com/YrrWhbE.png

* Mention that an elevator or travel anchor can also replace one of the floor blocks to be flush with it, instead of having to be inside of the cleanroom.
* The quest talks about using a diode to pass more than 1A of power into the cleanroom. Maybe it would be a good idea to add a diode to one of the optional tasks as well for indication, or for people who don't read all the text in full detail?

**Comment by AmberSherbet** [comment]
- [ ] MV and LV chapter icons appear to be swapped (as of 2.2.9 Dev-9), switch them around

**Comment by chochem** [comment]
> * [ ]  MV and LV chapter icons appear to be swapped (as of 2.2.9 Dev-9), switch them around

the icons are completely fine. which is unsurprising as nobody has touched that in years.

**Comment by AmberSherbet** [comment]
> > MV and LV chapter icons appear to be swapped (as of 2.2.9 Dev-9), switch them around
> 
> the icons are completely fine. which is unsurprising as nobody has touched that in years.

In that case why is LV an Aluminum Plate while MV is an LV Circuit?

**Comment by chochem** [comment]
its an HV circuit. those are the goals of the tiers

**Comment by tankobuman** [comment]
![スクリーンショット (25)](https://user-images.githubusercontent.com/90376144/219310419-8da04c14-f4c0-43e6-aced-9374e2a719b8.png)
AE2 version quantum storage cell can store 63 types of items

**Comment by the-emerald** [comment]
![Jdk-8 0 352 8-hotspot Screenshot 2023 02 16 - 20 09 26 00](https://user-images.githubusercontent.com/12746683/219476672-546e81b2-624f-48b5-8717-80588b9cf113.png)

This quest should accept all 6 plated variants (not just bronze). By the time we get to nukes, bronze, steel, titanium should be readily available!

**Comment by AbdielKavash** [comment]
Could you add something about the Catwalks mod into Building Better Bases? It is a really neat and simple mod that adds realistic-looking, well, catwalks, which are otherwise not really doable without microblock abuse and such. I didn't learn about it until today, it would be nice if there was a quest mentioning it.

I can even write the quest myself if asked (I just don't know how to actually add it to a PR).

**Comment by chochem** [comment]
> Could you add something about the Catwalks mod into Building Better Bases? It is a really neat and simple mod that adds realistic-looking, well, catwalks, which are otherwise not really doable without microblock abuse and such. I didn't learn about it until today, it would be nice if there was a quest mentioning it.
> 
> I can even write the quest myself if asked (I just don't know how to actually add it to a PR).

let me know when its done, I can put it in

**Comment by chochem** [comment]
> ![スクリーンショット (25)](https://user-images.githubusercontent.com/90376144/219310419-8da04c14-f4c0-43e6-aced-9374e2a719b8.png) Old EC2 quantum storage cell can store only single item but new AE2 one can store 63 types of items

looks more like a bug to me? not sure though

**Comment by AbdielKavash** [comment]
> let me know when its done, I can put it in

_Quest name:_ Catwalks

_Icon:_ Scaffold (from Catwalks Mod)

_Location:_ Building Better Bases tab. At the bottom of the page, to the right of "BetterIO?".

_Prerequisite:_ "Blastoff!" (Quest to make steel in Steam tab.)

_Tasks:_

1. Retrieval, not consumed:
4x Scaffold
6x Catwalk
6x Caged Ladder

2. Retrieval, not consumed, optional:
1x Blowtorch

_Reward:_
16x Steel Ingot

_(Note: The reward is enough to return the material cost of the tasks, if you craft all of them and use lathe for steel rods and screws. There are slightly more efficient assembler recipes for the items, and it is also possible to macerate them back into steel dust, which could make the quest a slight gain of steel, if anyone wants to bother with that. Nerf the reward if that is a problem.)_

_Quest text:_

> The _Catwalks Mod_ adds several cosmetic blocks to deck out your factory.
> 
> Scaffolds and Builder's Scaffolds are useful for building up in the air before you get access to flight. They can be climbed like ladders, and break instantly when punched. You will also not fall off of them if you let go of movement keys while climbing. If you right-click the Builder's Scaffold with a solid block, the scaffold will be replaced by that block.
> 
> Catwalks can connect to each other to form a wide passageway. You can decorate them by attaching Caution Tape or Rope Lights, the latter provide dim lighting.
> 
> Caged Ladders give you a way to move up and down without a risk of falling off. Climbing them is much faster than vanilla ladders.
> 
> Support Columns are purely decorative. They connect to each other in all directions. They also work as ladders.
> 
> Sturdy Rails let you run minecart tracks across gaps without supports. Right-click the Sturdy Powered Rail with a Blowtorch to power it permanently, you don't need a redstone torch. If you don't have enough distractions already, use them to build a rollercoaster!
> 
> The Blowtorch lets you remove railings or even the floor from a catwalk by right-clicking. Use it to make an opening to let you walk onto the catwalk, or if you don't know what an "OSHA violation" is. The same function can be performed by a GT or Forge-compatible wrench, so you don't actually need to make this.

**Comment by AbdielKavash** [comment]
The Convocation of the Damned quest in the Blood Magic questline requires an _unbound_ activation crystal to complete. If you have ever bound the activation crystal to yourself (for example, to activate any ritual), you will have to either craft a new crystal or perform the Ritual of Unbinding to complete the quest.

Would suggest making the quest work regardless of the owner of the crystal; or if this is impossible make the crystal task optional.

**Comment by Borjomm** [comment]
**Quest:** Some Rare High Tier Metals
**Tab:** HV
There is nitric acid in an optional task, but chalcopyrite and pentlandite processing now requires sulfuric acid, so just change it to sulfuric acid

**Comment by chochem** [comment]
- [ ] Quest: GT6-styled pipes: 
![image](https://user-images.githubusercontent.com/40274384/222939507-5a34b21f-4d5d-46a5-9c84-ab61b6184b64.png)
kind of shocked to read that. how long have we been spreading this nonsense?

**Comment by King65868** [comment]
Nethershard and Endshard bee quests have ignore NBT on, should have it off.

**Comment by King65868** [comment]
1k fluid storage cell quest says the fluid cell can hold 5 types of liquids, not true anymore. Also talks about storage with 5 types.

**Comment by chochem** [comment]
a few points discussed in dev-voice chat:
- EV progression change based on rocket change!
- remove old quests for oil (LV), bio (MV). after bq update.
- make new enderiotab from mv and hv
- add retroactive questcompletions for chem line skip quests. make each complete the requirements.
- post europium luv stuff is really zpm
- reorganize uhv tab + endgame tab.
- quest power up fluid nukes has broken task
- rebalance lootbags? nerf IV/AE2, buff others
- gold capped greatwood wand needs clear description+title
- 7 cores in adept -> single cores
- and even better conductor quest can be removed
- two quest after need a core
- last one should just point to kami-ha-me-ha tab.
- add quests for BM meteors
- new questtab for mobility (!!): teleportation, boats, elevator, jetpacks, horse

**Comment by glowredman** [comment]
- [ ] "And so it begins" tab, "Even more challenge needed" quest, 4th paragraph, last sentence: typo, should be "expect" instead of "exect"

**Comment by chochem** [comment]
- [ ] quest 'safety first': GT power loss was changed!

**Comment by King65868** [comment]
Soul bee quest says soul bees have the highest chance of making soul combs, but the dreaming bee now does.

**Comment by BritishCynic** [comment]
- [ ] Quest "Faster Crops" in 2.3.1 has no icon and a .name in the last retrieval task.

**Comment by chochem** [comment]
the eig has been fixed now. was a incorrect id, not the quest. (see https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/12978)

**Comment by chill-was-taken** [comment]
1. Quest Safety First
"Note that cables are batter than wires."
typo. batter -> better


2. The starting message of this thread:
"The most up-to-date quest files can always be found here"
The link after this phrase is broken.

**Comment by glowredman** [comment]
- [ ] "Walk Through Fire" quest completes with normal Pig Adventure Backpack

**Comment by chochem** [comment]
- [ ] remove OSS quest. its super buggy and not needed for anything.

**Comment by chochem** [comment]
- [ ] remove quests 'bzzz... Whoom (Defunct do not complete)'. its just some old remnant where someone didnt do a clean job of removing a quest.

**Comment by chochem** [comment]
closed in favour of https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/13036
